hoth_api.yaml

```diff
@@ -265,4 +293,14 @@ components:
           type: integer
         content:
           type: object
-          additionalProperties: true
\ No newline at end of file
+          additionalProperties: true
+    GenerateDocumentRequest:
+      oneOf:
+        - $ref: "#/components/schemas/DescriptionOfWork"
+        - $ref: "#/components/schemas/SomeFutureDocument"
+    DescriptionOfWork:
+      description: "TODO: AT-7341, define DescriptionOfWork schema"
+      type: object
+    SomeFutureDocument:
```

Comment:
We can just leave this off for now; I included it in the ticket just to make my intentions clear.